src/components/input/input.jsx

```diff
@@ -1,37 +1,54 @@
 import PropTypes from 'prop-types';
 import React from 'react';
+import cautionSvg from '../../svgs/icon-caution-fill.svg';
+import Icon from '../icon/icon.jsx';
 import styles from './input.css';
-import labelPropType from './label-prop-type';
 
-/**
- * Base input component. Accepts any prop that an input can take.
- * @see [RFC](https://github.com/mavenlink/rfc/pull/75)
- * @see [Design spec](https://zpl.io/295Yp7p)
- */
-export default function Input({ 'aria-labelledby': ariaLabelledBy, className, id, ...rest }) {
+function getClassName(className, invalid) {
+  if (className) return className;
+  return invalid ? styles['invalid-input'] : styles.input;
+}
+
+export default function Input(props) {
   return (
-    <input
-      {...rest}
-      aria-labelledby={ariaLabelledBy}
-      className={className}
-      id={id}
-    />
+    <div className={styles.container}>
+      <label htmlFor={props.id} className={styles.label}>
+        {props.label}
+        {props.required && <span>(Required)</span>}
```

Comment:
Looks like `(Required)` and `(Optional)` do not have any consistent logic due to the nature of our application.

Sometimes a form has all inputs as required inputs which means we do not want to clutter the UI with `(Required)`, opting for something else.

Same thing with `(Optional)`. 

Thus, I think we should not bake any logic about appending to the label and let the context of the `<Input>` decide on how the label looks.